test/storage/garbage_collector_test.cpp

```diff
@@ -905,10 +934,10 @@ TEST_F(GarbageCollectorTests, TwoTupleOLAP) {
     EXPECT_FALSE(tested.select_result_);
 
     txn_manager.Commit(txn0, TestCallbacks::EmptyCallback, nullptr);
-    // Unlink read-only txn 0 and txn 1, 3, 5, 7 as txn 0 committed
-    // Deallocate txn 2, 6
+    // Unlink read-only T0 and INSERT, U2, INSERT', U2' as T0 committed
+    // Deallocate U1, U1'
     EXPECT_EQ(std::make_pair(2u, 5u), gc.PerformGarbageCollection());
-    // Deallocate txn 1, 3, 5, 7
+    // Deallocate INSERT, U2, INSERT', U2'
     EXPECT_EQ(std::make_pair(4u, 0u), gc.PerformGarbageCollection());
   }
 }
```

Comment:
Another check here to make sure that nothing gets deallocated / unlinked after this?